src/api/app/views/webui/package/responsive_ux/_buildstatus.html.haml

```diff
@@ -0,0 +1,68 @@
+    = label_tag "show_all_#{index}" do
+      %u.custom-label #{label_message} excluded/disabled results
+
+- buildresults.results.each_pair do |package, results|
+  :ruby
+    package_name = package.tr('.:', '_')
+    expanded = collapsed_packages.exclude?(package_name)
+  %h5.d-flex.flex-row.text-primary.mt-3.mb-3
+    = package
+    - if buildresults.results.count > 1
+      = collapse_link(expanded, package_name)
+  .collapse#package-buildstatus{ class: "collapse-#{package_name}#{expanded ? ' show' : ''}", data: { main: package_name } }
+    - if results.present?
+      - previous_repo = nil
+      - results.each do |result|
+        :ruby
+          repository_name = result.repository.tr('.', '_')
+          expanded = repository_expanded?(collapsed_repositories, repository_name, package_name)
+        - if result.repository != previous_repo
+          .d-flex.flex-row.py-1.bg-light.pl-1.pl-sm-2
+            = link_to(word_break(result.repository, 22),
+              package_binaries_path(project: project, package: package, repository: result.repository),
+              title: "Binaries for #{result.repository}")
+            = collapse_link(expanded, package_name, repository_name)
+
+        .collapse{ class: "collapse-#{package_name}-#{repository_name}#{expanded ? ' show' : ''}",
+                   data: { repository: repository_name, main: package_name } }
+          .d-flex.flex-row.flex-wrap.pt-1
+            .repository-state
+              - if !result.is_repository_in_db
+                %i.fas.fa-clock.text-warning{ title: 'This result is outdated' }
+              - else
+                = repository_status_icon(status: result.state)
+              %span.ml-1
+                = result.architecture
+            .build-state.toggle-build-info-parent
+              %i.fa.fa-question-circle.text-info.px-2.pl-lg-1.toggle-build-info{ title: 'More information' }
+              = arch_repo_table_cell(result.repository, result.architecture, package, 'code' => result.code, 'details' => result.details)
+            .build-info.mt-1.ml-3.mb-3.mr-3.collapsed
+              .triangle.center
+              .build-info-content
+                %p.py-1= Buildresult.status_description(result.code)
+                %div
+                  - if !result.is_repository_in_db
+                    %i.fas.fa-clock.text-warning
+                    This result is outdated
+                  - else
+                    = repository_status_icon(status: result.state)
+                    %span.pl-1= repository_info(result.state)
+                - if result.details
+                  %div
+                    %strong #{result.code}:
+                    %span= result.details
+
+        - previous_repo = result.repository
+    - else
+      All the results have state
+      %strong excluded/disabled
+
+:javascript
+  $('.toggle-build-info').on('click', function(){
```

Comment:
@DavidKang this javascript code exists both in the ```build_results``` partials for package and project. Maybe you can put it into a function in the assets, and call it from there :)

Reply:
Besides that the PR looks good to me